example/client.rb

```diff
@@ -68,7 +68,9 @@
 
 stream.on(:close) do
   log.info 'stream closed'
-  sock.close
+
+  # keep connection open to handle push messages
+  # sock.close
```

Comment:
OK, I guess that's fine -- it's just a demo server. Can you drop the comments though? My issue is what the message indirectly implies: just because the connection is open doesn't mean you can handle push messages.

If we drop both lines then we're good.